README.md

```diff
@@ -23,6 +23,13 @@ daemon that can be controlled by the AS is nginx
 * [ETSI TS 126 512](https://portal.etsi.org/webapp/workprogram/Report_WorkItem.asp?WKI_ID=66919) - 5G Media Streaming (
   5GMS): Protocols (3GPP TS 26.512 version 17.1.2 Release 17)
 
+## Install dependencies
+
+```
+sudo apt install git python3-pip python3-venv
+pip install build
```

Comment:
The recommendation is to use `python3 -m pip` instead of just `pip` to ensure you are using the python binary you want to install under.

Reply:
You misunderstood: `pip install build` should be `python3 -m pip install build`

Reply:
aaah sorry, let me fix it

Reply:
Done